cmd/utils/flags.go

```diff
@@ -1895,7 +1895,9 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 		}
 		cfg.Genesis = core.DefaultStoryGenesisBlock()
 		SetDNSDiscoveryDefaults(cfg, params.StoryGenesisHash)
-		cfg.Miner.GasPrice = big.NewInt(params.GWei * 16)
+		cfg.Miner.GasPrice = big.NewInt(params.GWei * 4)
+		cfg.Miner.GasCeil = 45_000_000
+		cfg.TxPool.NoLocals = true
```

Comment:
Why enable this by default? This is for price exemptions for locally submitted transactions